vendor/popover_menu.js

```diff
@@ -0,0 +1,74 @@
++function ($) {
```

Comment:
hmm if we only need this for tests (in this repo), we could also just get a stub of it (and bootstrap) for now.  That may be the way to go here